src/_data/plugins/eleventy-plugin-broken-links.json

```diff
@@ -1,5 +1,5 @@
 {
   "npm": "eleventy-plugin-broken-links",
-  "author": "bburgess_keys",
-  "descrption": "A plugin to check your build for broken external links and redirects"
+  "author": "bradleyburgess",
```

Comment:
This should actually be a Twitter username.

Reply:
@pdehaan Thanks. I did a little digging, though, and it seems like we have a choice, as per [this line in the `.eleventy.js` config file](https://github.com/11ty/11ty-website/blob/7e3e529f1a64d579683bb58242d9df2eccff7fd6/.eleventy.js#L32). If the `author` property startes with `twitter:`, then it fetches your Twitter avatar; if not it gets GitHub. Does that seem right? I really wish my two handles were the same 🤦

Reply:
🤷  maybe.
Historically it has always been Twitter usernames. But I think the logic changed slightly once we moved the "sites/*.json" from this repo to the 11ty-community "built-with-eleventy/*.json" repo and some of those records had some legacy/fallback "twitter:" prefixed records to disambiguate between GitHub and Twitter usernames. But sure, you could try with a "twitter:" prefix if you want. I guess the worst that could happen is that when the site is deployed next it may or may not link an avatar. #yolo

But yeah, I think the ultimate goal is to eventually (no ETA) remove all these plugins/*.json files and fetch them from an upstream API. Presumably npm directly, if it's easy; or npms.io if it's reasonably in sync w/ npm's registry.

Reply:
Thanks for that explanation. I did see in the `11ty/11ty-community` repo that there is a plan to move to sourcing the plugins from an API. 

In the mean time, it seems like without the `twitter:` prefix, it fetches GitHub, evidenced by the blank / default GitHub avatar instead of my Twitter avatar:

![Screenshot_20220424-105029.png](https://user-images.githubusercontent.com/47909061/164968530-67b69252-4c83-4be5-89e4-3e318a08a3e8.png)

So I think I will use my GitHub handle for now, and when things change I'll update whatever needs updating 🤷‍♂️

So this PR is accurate as it is I think. Again, appreciate your help / explanation.